synapse/lib/layer.py

```diff
@@ -1076,6 +1076,24 @@ def getAbrvProp(self, abrv):
 
         return s_msgpack.un(byts)
 
+    async def getFormInByProp(self, buid):
```

Comment:
Is this method going to be used exclusively internally? If so, we should implement with a _ so epiphyte doesn't plumb it through the cluster layer :)

Reply:
If its only ever called by the *layer* itself I shouldn't need to touch it. The real problems arise from *external* callers to layer APIs.